services/shhext/api.go

```diff
@@ -65,6 +66,9 @@ type MessagesRequest struct {
 	// Topic is a regular Whisper topic.
 	Topic whisper.TopicType `json:"topic"`
 
+	// Topics is a list of Whisper topics.
+	Topics []whisper.TopicType `json:"topics"`
```

Comment:
`Topic whisper.TopicType` should be marked as deprecated. We should limit the number of parameters.

Reply:
@adambabik good idea, is there a way to do it automatically or do you mean logging it?

Reply:
Just add a comment:
```
// Topic is a regular Whisper topic.
// DEPRECATED
```

There is no way we can tell the client about that.